packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -886,6 +887,41 @@ function* watchRestoreOriginalEngineResults() {
   });
 }
 
+function* emitEntityUpdatedEvent(tdoId) {
+  const config = yield select(configModule.getConfig);
+  const { apiRoot, graphQLEndpoint } = config;
+  const graphQLUrl = `${apiRoot}/${graphQLEndpoint}`;
+  const token = yield select(authModule.selectSessionToken);
+
+  // TODO: remove hardcoded CMS applicationId when platform starts using orgId from the token
+  const input = {
+    eventName: 'EntityUpdated',
+    eventType: 'entity',
+    application: '8a37c1d0-3f3b-48d0-a84e-2b8e3646fbe5',
+    payload: `{ "recordingId": "${tdoId}" }`
+  };
+
+  try {
+    if (tdoId) {
+      const response = yield call(fetchGraphQLApi, {
```

Comment:
Can we use callGraphQLApi here? https://github.com/veritone/veritone-sdk/blob/fdd6201b9e849b331c1bb436af0dffc60b2505f6/packages/veritone-widgets/src/redux/modules/engineOutputExport/index.js#L424

Reply:
Thank you. I'll address this in a separate commit.